app/serializers/atmosphere/appliance_type_serializer.rb

```diff
@@ -37,8 +36,12 @@ def vmt_with_state?(state)
           count > 0
     end
 
-    def tenants
-      object.tenants.active
+    def compute_site_ids
+      ts = object.tenants.active
+      if !(@options[:load_all?])
```

Comment:
I don't like unless. Leads to awkward statements (such as "unless-else"). I try to avoid it whenever possible, but fine, I will humor your request. ;)